docs/src/examples/components/Text/Types/TextWeightsExample.shorthand.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Text } from '@stardust-ui/react'
+
+const TextWeightsExampleShorthand = () => (
+  <div>
+    <div>
+      <Text weight="light">This text is light.</Text>
+    </div>
+    <div>
+      <Text weight="semilight">This text is semilight.</Text>
+    </div>
+    <div>
+      <Text weight="regular">This text is regular.</Text>
+    </div>
+    <div>
+      <Text weight="semibold">This text is semibold.</Text>
+    </div>
+    <div>
+      <Text weight="bold">This text is bold.</Text>
+    </div>
```

Comment:
We strive for brevity in public facing examples in order to save on vertical screen real estate and making it easier to quickly visually scan the code.  How about using `<br />` for new lines to save on elements and lines of code?

```jsx
  <div>
    <Text weight="light">This text is light.</Text> <br />
    <Text weight="semilight">This text is semilight.</Text> <br />
    <Text weight="regular">This text is regular.</Text> <br />
    <Text weight="semibold">This text is semibold.</Text> <br />
    <Text weight="bold">This text is bold.</Text>
  </div>
```

Reply:
Understood. I'll also apply the same to the Sizes example while I'm at it.